bench/benches/packets.rs

```diff
@@ -219,11 +217,8 @@ fn single_push(c: &mut Criterion) {
 #[nb2::bench(mempool_capacity = 511)]
 fn multi_push(c: &mut Criterion) {
     c.bench_function("packets::multi_push_udp", |b| {
-        b.iter_batched(
-            || Mbuf::new().unwrap(),
-            multi_push_udp,
-            BatchSize::NumIterations(BATCH_SIZE as u64),
-        )
+        let s = v4_udp().prop_map(|_v| Mbuf::new().unwrap());
```

Comment:
I think the old one is better. here a bunch of v4 udp packets are generated then thrown away not used at all. seems pointless and confusing. the old one just creates new mbufs.

Reply:
So. It was different as more iterations were used then the iter_proprest ones. That may be fine for this case I guess since no comparison.

Reply:
after discussion, will make it where we can gen an mbuf as a strat.

Reply:
ok discussed off line for a slight change to the new code. make `any::<Mbuf>` strategy work and use the new proptest batch. 👍